hexa/user_management/graphql/schema.graphql

```diff
@@ -235,6 +235,16 @@ input OrganizationInput {
     url: String
     contactInfo: String
 }
+
+type FeatureFlag {
+    code: String!
+    config: JSON!
+}
+
+extend type Me {
```

Comment:
`Me` is defined just a few lines above - why extend it?

Reply:
Because I didn't see it 🙈